release/nightly_tests/dataset/multi_node_train_16_workers.yaml

```diff
@@ -0,0 +1,15 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 1
+
+head_node_type:
+    name: head_node
+    instance_type: m5.4xlarge
```

Comment:
I think there is a way to pass a custom num_gpus here, so that way we can still use --use-gpu for the script.